ape_safe.py

```diff
@@ -34,15 +34,15 @@
     1: 'https://safe-transaction.mainnet.gnosis.io',
     4: 'https://safe-transaction.rinkeby.gnosis.io',
     5: 'https://safe-transaction.goerli.gnosis.io',
-    10: 'https://safe-transaction.optimism.gnosis.io',
+    10: 'https://safe-transaction-optimism.safe.global',
     56: 'https://safe-transaction.bsc.gnosis.io',
     100: 'https://safe-transaction.xdai.gnosis.io',
     137: 'https://safe-transaction.polygon.gnosis.io',
```

Comment:
```suggestion
    137: 'https://safe-transaction-polygon.safe.global',
```
All networks updates have completed as per the schedule [here](https://forum.gnosis-safe.io/t/transaction-service-migration-october-2022/1550)